demo/chbench/mzcompose.yml

```diff
@@ -343,9 +344,12 @@ mzworkflows:
   # Stops the load generator and source database but leaves the rest of the cluster running
   prepare-snapshot:
     env:
-      MZ_CHBENCH_SNAPSHOT: ${MZ_CHBENCH_SNAPSHOT:-./tmp}
-      MZ_CHBENCH_SNAPSHOT_KAFKA_DATA: ${MZ_CHBENCH_SNAPSHOT:-./tmp}/kafka-data
-      MZ_CHBENCH_SNAPSHOT_ZK_DATA: ${MZ_CHBENCH_SNAPSHOT:-./tmp}/zookeeper
+      # If MZ_CHBENCH_SNAPSHOT is defined, use a sub-directory within the snapshot directory
+      # Otherwise, use a named volume
+      # Docker compose doesn't know how to interpolate using ":+", so the interpolation must be
+      # done here, using mzcompose
+      MZ_CHBENCH_SNAPSHOT_KAFKA_DATA: ${MZ_CHBENCH_SNAPSHOT:-kafka-data}${MZ_CHBENCH_SNAPSHOT:+/kafka-data}
```

Comment:
should this not be `./kafka-data` (leading-dot) to make it a sub-directory?

Reply:
Which part? The first one should be the name of a volume and the second one should be an extension to the path, not a path relative to the current working directory.